### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Dragoshiz**
added login form and endpoint, also refactored a little bit the register page with how its handling errors, and other error messages

### CODE REVIEW

File: web/django/backend/__pycache__/utils.cpython-310.pyc
Comment (pandaero) [inline web/django/backend/__pycache__/utils.cpython-310.pyc:1]: we should add *.pyc to the .gitignore

File: web/django/backend/auth_app/migrations/0001_initial.py
Comment (pandaero) [inline web/django/backend/auth_app/migrations/0001_initial.py:1]: generated files should be in .gitignore

**Review by pandaero — CHANGES_REQUESTED** [review]
Let's modify the .gitignore to exclude the binary .pyc and generated Django files

**Comment by Dragoshiz** [comment]
thank you

**Pull request closed without merging**